auth-web/src/views/auth/HomeView.vue

```diff
@@ -271,8 +284,11 @@ export default class HomeView extends Vue {
     return !!this.currentAccountSettings
   }
 
-  private goToManageBusinesses (): void {
-    this.$router.push(`/${Pages.MAIN}/${this.currentAccountSettings.id}`)
+  private goToManageBusinesses (isNcRequest: boolean = false): void {
```

Comment:
Just a minor comment.. I hope Nc is for NameChange.. do you mind renaming the variable to make it more clear ? something like IsNameChangeRequest or IsNameChangeReq...the abbreviation Nc takes a moment to understand..

Reply:
sorry, its actually numbered company request! 100%, i will rename this to make it more clear :)

Reply:
thanks..its actually beautifully written code .loved it..

Reply:
thanks a lot! The feedback is really appreciated as well 👍